nixos/modules/services/misc/greenclip.nix

```diff
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.greenclip;
+in {
+
+  options.services.greenclip = {
+    enable = mkEnableOption "Whether to enable the greenclip daemon that will listen to selections";
```

Comment:
mkEnableOption is defined as ```
  mkEnableOption =
    # Name for the created option
    name: mkOption {
    default = false;
    example = true;
    description = "Whether to enable ${name}.";
    type = lib.types.bool;
  };

```
so the current value is not good. "Greenclip daemon" should be enough ?

Reply:
@teto True, just saw the mkEnableOption source. Fixed.